modules/ROOT/pages/install-uninstall-reinstall.adoc

```diff
@@ -3,80 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-If you encounter problems during installation, you can uninstall. Before beginning a new installation, ensure that your environment meets all system requirements listed in the xref:install-checklist.adoc[Product Prerequisites], and that all remnants of the original installation have been removed by performing the following commands on all nodes:
+If you encounter problems during installation, you can uninstall. Before beginning a new installation, ensure that your environment meets all system requirements listed in the xref:install-checklist.adoc[Product Prerequisites].
```

Comment:
Should we remove this page altogether? The uninstall instructions are meant for when the installation is problematic, so this should be part of the troubleshooting guide for Professional Services. Since this will be done by Professional Services, it may not need to be in public docs.

Reply:
@federicoboerr - Do you agree?

Reply:
+1 on removing this whole section.
This page has already been moved to confluence.

Reply:
Removed content and left a blurb similar to the install one.

Reply:
Suggestion: at https://beta.docs.stgx.mulesoft.com/beta-pce-21/private-cloud/2.1/install-workflow instead of calling it an Install section, how about renaming it to "Getting Ready for Installation", have the same content in it. Because for me, it's kind of misleading that is says "Install" but there are no Installation steps, rather indications on how to make sure you're ready to install.

Reply:
Makes sense - I've updated the title.